docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
@nickpan47 
>  job.io.system also implicitly forces input and output systems into one

I agree that it should force the input and output into one. But calling it "default" doesn't say that it will not be used for metadata system. From the documentation here, I infer that this configuration will be used for "_any input or output streams for which the system is not explicitly configured_". We are defining a separate "default" system for metadata topics, are we? 

Also, are we going to use "job.coordinator.system" for all samza metadata system or just coordinator and checkpoint?